requirements.txt

```diff
@@ -3,5 +3,5 @@ sip==4.19.7
 requests>=2.22.0
 psutil>=5.4.7
 behave>=1.2.6
-fbs==0.6.6
+fbs==0.8.3
```

Comment:
I guess we don't. Well, it is not a bad idea to keep it for a while. As soon as in this version, we allows user using `fbs run` under `virtualenv`, it might need `requirements.txt` to install packages.